crates/starknet-replay/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "starknet-replay"
```

Comment:
there are already 2 existing crates/projects called `starknet-replay`, I'd suggest not creating a 3rd ;)

Reply:
I need a better name :(

Reply:
maybe `rpc-replay`? not great, but at least it's unique

Reply:
Sold!